assignments/timed-assignment-rest/Photo-Contest|Assignment-7.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Photo Contest</title>
+    </head>
+    <body>
+        <h1>This is a photo contest</h1>
+
+        <form>
+            <label for="fullNameInput">Full Name</label>
+            <input type="text" name="fullName" id="fullNameInput" required />
+
+            <label for="emailInput">Email</label>
+            <input type="email" name="email" id="emailInput" required />
+
+            <label for="photoInput">Upload a photo</label>
+            <input
+                type="file"
+                name="photo"
+                id="photoInput"
+                accept="images/*"
+                required
+            />
+
+            <label for="">I agree to terms and conditions</label>
+            <input type="checkbox" required />
+
+            <label for="countrySelect">Select your Country</label>
+            <select name="contestantCountry" id="countrySelect">
+                <option value="United States">United States</option>
+                <option value="Canada">Canada</option>
+                <option value="United Kingdom">United Kingdom</option>
+                <option value="Australia">Australia</option>
+            </select>
+
+            <label for="descriptionTextarea"
+                >Write a brief description or story about your photo</label
+            >
+            <textarea
+                name="descriptionOfThePhoto"
```

Comment:
just "description" is better choice for name attr

Reply:
keep photoDescription